pkgs/applications/misc/nwg-displays/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, fetchFromGitHub
+, atk
+, gdk-pixbuf
+, gobject-introspection
+, gtk-layer-shell
+, gtk3
+, pango
+, python310Packages
+, wrapGAppsHook
+}:
+
+python310Packages.buildPythonApplication rec {
+  pname = "nwg-displays";
+  version = "0.3.7";
+
+  src = fetchFromGitHub {
+    owner = "nwg-piotr";
+    repo = "nwg-displays";
+    rev = "v${version}";
+    hash = "sha256-Y405ZeOSpc1aPKEzFdvlgJgpGAi9HUR+Hvx63uYdp88=";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    gtk3
+  ];
+
+  propagatedBuildInputs = [
+    atk
+    gdk-pixbuf
+    gtk-layer-shell
```

Comment:
Sort :)

Reply:
Could you please make that last change?

Reply:
? i did run sort on that

Reply:
the last force push https://github.com/NixOS/nixpkgs/compare/ce77f99f8f924d95f001dd8592259e57827fc3a0..662712806f997ead0176b3f797c2fb00ed9e35c6

Reply:
Weird, gdk-pixbuf should come after... But anyway, the pr LGTM, I'll merge it when ci is ok or tomorrow.

Reply:
well, i just sorted alphabetically so it would do ```d``` then ```t```. sounds good, thanks again drupol. appreciate it

Reply:
Ooooh it gdk and gtk.... Sorry man, I need some sleep 😴